test/controllers/public_recipes_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PublicRecipesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
- [ ] Please work on testing your controllers accordingly and remove any unnecessary/commented code.